packages/core/core/src/core-api/controller/index.ts

```diff
@@ -59,6 +59,7 @@ function createController({
       return strapi.contentAPI.validate.query(ctx.query, contentType, { auth });
     },
 
+    // TODO: should this just accept ctx and pull body.data from it?
```

Comment:
We might want to validate other data than ctx.body, I think it makes sense to have both parameters